src/main/scala/com/sumologic/sumobot/core/model/Messages.scala

```diff
@@ -21,9 +21,16 @@ package com.sumologic.sumobot.core.model
 import java.io.File
 
 import akka.actor.ActorRef
+import slack.models.Attachment
```

Comment:
This pollutes the internal model with classes from the underlying Slack client. Can it easily be avoided?